### PR TITLE
Update Pressbooks to PHP 7.3 and WordPress 5.5.3

### PR DESCRIPTION
**Pull request by SteelWagstaff**
This PR bumps the PB version to 5.17.4-dev and updates Pressbooks to require PHP 7.4 and WordPress 5.5.3 as minimum versions. It also removes previous versions of PHP from the test matrix used by Travis.

I used https://github.com/pressbooks/pressbooks/commit/15f946b1a976bf0082d50bbcc33047f2a9be0679 as a reference to understand what changes needed to be made.

Fix for https://github.com/pressbooks/pressbooks/issues/2067

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 === Pressbooks ===
 Contributors: Pressbooks <code@pressbooks.com>
 Tags: ebooks, publishing, webbooks
-Requires at least: 5.4
-Tested up to: 5.5.2
-Requires PHP: 7.1
+Requires at least: 5.5.3
```
Comment (ho-man-chan) [inline readme.txt:4]: I believe we can set the requires at least to 5.5 so users don't get a nag if their wordpress is not up-to-date.

**Review by ho-man-chan — CHANGES_REQUESTED** [review]
looks good. I think we can lower the request at least to 5.5 for the wp version.

File: readme.txt (hunk 1 above)
Comment (SteelWagstaff) [reply readme.txt:4]: I'd prefer to set it to 5.5.3 as it's a stable release and the one we use in our production bedrocks.

**Review by ho-man-chan — APPROVED** [review]
LGTM

**Pull request merged**